apps/aecore/src/aec_block_generator.erl

```diff
@@ -11,7 +11,10 @@
 
 -export([get_candidate/0]).
 
--export([start_generation/0, stop_generation/0]).
+-export([ get_generation_state/0
```

Comment:
Should this be a test only exported function?

Reply:
Yes, it might be. i didn't do it because it corresponds to `aec_conductor:get_mining_status/0` which is not hidden.

I don't have strong opinion on that, I can hide it if requested